Tests/ImagineEngineTests/TextureManagerTests.swift

```diff
@@ -21,21 +22,37 @@ class TextureManagerTests: XCTestCase {
     func testFallsBackToLowerScaleTextures() {
         _ = manager.load(Texture(name: "texture"), namePrefix: nil, scale: 3)
 
-        XCTAssertEqual(imageLoader.imageNames, ["texture@3x", "texture@2x", "texture"])
+        XCTAssertEqual(imageLoader.imageNames, ["texture@3x.png", "texture@2x.png", "texture.png"])
     }
 
     func testRemembersTextureScaleFallback() {
-        imageLoader.images["texture@2x"] = makeImage()
+        let textureToLoad = Texture(name: "texture")
+        imageLoader.images["texture@2x.png"] = makeImage()
 
-        _ = manager.load(Texture(name: "texture"), namePrefix: nil, scale: 3)
-        XCTAssertEqual(imageLoader.imageNames, ["texture@3x", "texture@2x"])
+        _ = manager.load(textureToLoad, namePrefix: nil, scale: 3)
+        XCTAssertEqual(imageLoader.imageNames, ["texture@3x.png", "texture@2x.png"])
 
         imageLoader.clearImageNames()
 
-        _ = manager.load(Texture(name: "texture"), namePrefix: nil, scale: 3)
+        _ = manager.load(textureToLoad, namePrefix: nil, scale: 3)
         XCTAssert(imageLoader.imageNames.isEmpty)
     }
 
+    func testUsesCorrectFormatWhenLoadingTexture() {
+        let pngTexture = Texture(name: "texture", format: .png)
+        let jpgTexture = Texture(name: "texture", format: .jpg)
+        let pngImage = makeImage()
+        let jpgImage = makeImage()
+        imageLoader.images["texture.png"] = pngImage
+        imageLoader.images["texture.jpg"] = jpgImage
+
+        let loadedPNGTexture = manager.load(pngTexture, namePrefix: nil, scale: 1)!
```

Comment:
You don't really need to force-unwrap here, since the equality check will fail if it's `nil`. The advantage is that we get a much nicer error message in the CI logs in case this ever starts returning `nil`.